docs/source/bricks.rst

```diff
@@ -107,6 +111,38 @@ Examples:
       text = f.read()
   elements = partition_email(text=text)
 
+  with open("example-docs/fake-email.eml", "r", content_source="text/plain") as f:
+      text = f.read()
+  elements = partition_email(text=text)
+
+  with open("example-docs/fake-email.eml", "r", get_meta_data=True) as f:
+      text = f.read()
+  elements = partition_email(text=text)
+
+
+``partition_text``
+---------------------
+
+The ``partition_text`` function partitions text files. The ``partition_text`` 
+takes a filename, file-like object, raw text, or a list of string objects as input and produces a list of
```

Comment:
Reminder to take the "or a list of string objects" out of the docs now too since that option's removed.